src/components/GoFormGenerator/GoFormGroup.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div :class="getFieldRowClasses">
+    <label
+      v-if="hasLabel"
+      :for="field.id"
+      :class="field.labelClasses"
+      class="control-label m-b-xs"
+    >
+      <span v-html="field.label" />
```

Comment:
The use of v-html seems to be not the best solution. 
https://forum.vuejs.org/t/performance-v-html-vs-interpolation-aka-braces/29559/5

Why not using {{ field.label }} ? To have an autoclose element I guess ?